fastlane/Constants/Constant.swift

```diff
@@ -28,6 +28,12 @@ enum Constant {
     static let appleProductionTeamId = "<#teamId#>"
     static let keychainName = "{PROJECT_NAME}_keychain"
     static let matchURL = "git@github.com:{organization}/{repo}.git"
+    static let apiKey: [String: Any] = [
+        "key_id" : EnvironmentParser.string(key: "API_KEY_ID"),
+        "issuer_id": EnvironmentParser.string(key: "ISSUER_ID"),
+        "key": EnvironmentParser.string(key: "APPSTORE_CONNECT_API_KEY"),
+        "in_house": false
+    ]
```

Comment:
Should it be?
```suggestion
    static let apiKey: [String: Any] = [
        "key_id" : Secret.appStoreKeyIdKey,
        "issuer_id": Secret.appStoreIssuerIdKey,
        "key": Secret.appstoreConnectAPIKey,
        "in_house": false
    ]
```

Reply:
I updated it at https://github.com/nimblehq/ios-templates/pull/540/commits/dd1965a8f0535d92ac74fe697538cc4dd01dab29 🙏